gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```

Comment:
Do not update this manually. Always call the task wrapper with appropriate arguments to update. That will also update the wrapper jar and executables.

Reply:
The jar and executables didn't change from 4.4 to 4.5.

Reply:
the jar did change. made a PR to update it: https://github.com/JakeWharton/butterknife/pull/1191

Reply:
It didn't in other projects I've upgraded

Reply:
Could be that someone forgot to change it when going from 4.x to 4.4.1